ksql-version-metrics-client/src/main/java/io/confluent/ksql/version/metrics/KsqlVersionCheckerAgent.java

```diff
@@ -78,6 +81,12 @@ public void start(final KsqlModuleType moduleType, final Properties ksqlProperti
 
   }
 
+  @Override
+  public ActiveChecker getActiveChecker() {
```

Comment:
This can just update some internal long:

```suggestion
  public void updateLastRequestTime() {
     lastRequestTimestamp = System.currentTimeMillis();
  }
```